test/test_sdk.py

```
@@ -0,0 +1,162 @@
+
+        with self.assertRaises(TypeError):
+            Client(client_config, http_client)
+
+    def test_sdk_create_missing_api_key(self):
+        client_config = ClientConfig()
+        client_config.endpoint = "https://sandbox.test.io/v1"
+        client_config.application = "website"
+        client_config.environment = "dev"
+
+        http_client = DefaultHTTPClient(DefaultHTTPClientConfig())
+        client = Client(client_config, http_client)
```

Ruff (0.15.2)

[error] Local variable `client` is assigned to but never used

Remove assignment to unused variable `client`

(F841)